EventPlanning.service/Startup.cs

```diff
@@ -33,6 +36,32 @@ public void ConfigureServices(IServiceCollection services)
 
             services.AddSingleton<ISmtpClient>(x => new SmtpWrapper("smtp.gmail.com", 587, smtpUser, smtpPass, true));
 
+            var jwtIssuer = Environment.GetEnvironmentVariable("JWTIssuer");
+            var jwtAudience = Environment.GetEnvironmentVariable("JWTAudience");
+            var jwtKey = Environment.GetEnvironmentVariable("JWTKey");
+            var jwtExp = Environment.GetEnvironmentVariable("JWTExpInMin");
+
+            if (jwtIssuer == null || jwtAudience == null || jwtKey == null || jwtExp == null 
+                || !Int32.TryParse(jwtExp, out var jwtExpAfter))
+            {
+                throw new Exception("JWT properties not provided");
+            }
+
+            services.AddAuthentication(JwtBearerDefaults.AuthenticationScheme).AddJwtBearer(options =>
+            {
+                options.TokenValidationParameters = new TokenValidationParameters
+                {
+                    ClockSkew = TimeSpan.FromMinutes(jwtExpAfter),
+                    ValidateIssuer = true,
+                    ValidateAudience = true,
+                    ValidateLifetime = true,
+                    ValidateIssuerSigningKey = true,
+                    ValidIssuer = jwtIssuer,
+                    ValidAudience = jwtAudience,
+                    IssuerSigningKey = new SymmetricSecurityKey(Encoding.UTF8.GetBytes(jwtKey))
+                };
+            });
```

Comment:
I would say that this should be extracted to a separate method called. Because start up method becomes clutered